frontend/model/contracts/shared/constants.js

```diff
@@ -132,3 +132,5 @@ export const POLL_STATUS = {
 }
 
 export const POLL_MAX_OPTIONS = 20
+export const POLL_QUESTION_MAX_CHARS = 280
+export const POLL_OPTION_MAX_CHARS = 280
```

Comment:
`280` here is a vallue I chose arbitrary from a existing value below:

```js
export const CHATROOM_DESCRIPTION_LIMITS_IN_CHARS = 280
```

Please let me know if we want to change this value.